src/Components/Header/Header.js

```diff
@@ -1,13 +1,20 @@
 import React from 'react';
 
-import logo from '../../assets/Images/logo.png';
+import logo from '../../assets/images/logo.png';
+import { FadeInTop } from '../../common/Animations/Animations';
```

Comment:
can't those paths be resolved to ./assets/etc...    and ./common/animations..... ??